v0/destinations/af/deleteUsers.js

```diff
@@ -0,0 +1,90 @@
+const { _ } = require("lodash");
+const { httpPOST } = require("../../../adapters/network");
+const { generateUUID } = require("../../util");
+const ErrorBuilder = require("../../util/error");
+
+/**
+ * This function will help to delete the users one by one from the userAttributes array.
+ * @param {*} userAttributes Array of objects with userId, emaail and phone
+ * @param {*} config Destination.Config provided in dashboard
+ * @returns
+ */
+const userDeletionHandler = async (userAttributes, config) => {
+  if (!Array.isArray(userAttributes)) {
+    throw new ErrorBuilder()
+      .setMessage("userAttributes is not an array")
+      .setStatus(400)
+      .build();
+  }
+  if (!config?.apiToken || !(config?.appleAppId || config?.androidAppId)) {
+    throw new ErrorBuilder()
+      .setMessage(
+        "API Token and Apple ID or Android App Id are required fields"
+      )
+      .setStatus(400)
+      .build();
+  }
+  const identityTypes = {
```

Comment:
Can we refactor this code its not pretty clear on what is format for userAttributes we are expecting and what we are setting

Reply:
Updated the logic here.